backend/account/serializer.py

```diff
@@ -109,3 +109,10 @@ def validate_password(self, value: Optional[str]) -> str:
                 "Password must be at least 3 characters long."
             )
         return value
+
+
+class UserSessionResponseSerializer(serializers.Serializer):
+    id = serializers.CharField()
```

Comment:
@muhammad-ali-e shouldn't this be a `UUIDField()`?

Reply:
Actually we need IntegerFeild here. Changed